docs/en/enterprise-edition/content-collections/runtime-security/vulnerability-management/scan-process.adoc

```diff
@@ -75,6 +75,21 @@ The following table lists the manifest files known to the scanner.
 
 The Console manages the current scan state and distributes the work to Defenders.
 
+[#go-symbol-extraction]
+=== Go Symbol Extraction
+When scanning Golang binaries, Prisma Cloud extracts Go symbols by default. This enables accurate vulnerability evaluation for specific package names and symbols. 
+
+Prisma Cloud also provides the ability to selectively disable symbol extraction, when unnecessary, to optimize the scanning process.
+
+To disable Go symbol extraction, complete these steps:
```

Comment:
@Kamesh-PaloAlto instead of ", complete these steps" add "one of the following options, as relevant:"